components/ledger/ledger-persistence/src/test/kotlin/net/corda/ledger/persistence/processor/impl/LedgerPersistenceRequestProcessorTest.kt

```
@@ -58,8 +63,7 @@ class LedgerPersistenceRequestProcessorTest {
         }
     }
 
-    @BeforeEach
-    fun setup() {
+    private fun defaultSetup() {
```

Why not keep the `@BeforeEach` and override the required mocks to throw the needed exception instead of using `defaultSetup()`?.